src/core/components/hv-root/index.tsx

```diff
@@ -524,6 +547,15 @@ export default class Hyperview extends PureComponent<Types.Props> {
         getRoot,
         updateRoot,
       );
+      Logging.info(
+        `[behavior] ${action} triggered with attributes: ${Logging.deferredToString(
```

Comment:
same here, the interpolation will happen before the Logging.info method is called, the call to `deferredToString` should be moved into a separate argument